va/va.go

```diff
@@ -481,6 +480,16 @@ func (va *ValidationAuthorityImpl) performRemoteValidation(
 	for _, i := range rand.Perm(remoteVACount) {
 		go func(rva RemoteVA) {
 			res, err := rva.PerformValidation(subCtx, req)
+			if res == nil {
+				responses <- &response{rva.Address, rva.Perspective, rva.RIR, res, err}
+				return
+			}
+			if res.Perspective != rva.Perspective || res.Rir != rva.RIR {
+				err = fmt.Errorf(
```

Comment:
I'd prefer it if this wrote the error to the responses channel and returned directly. As-is, line 493 is pulling double duty as sad path and happy path (and returning both a non-nil res and a non-nil err at the same time!) which I find slightly confusing.